ygot/diff.go

```diff
@@ -365,6 +365,9 @@ type DiffPathOpt struct {
 	// generated structs, which can result in duplication of list key leaves in
 	// the diff output.
 	MapToSinglePath bool
+	// OriginalPathsOnly specifies that only paths in the original value should be
+	// considered - updates that set unset fields will not be included in the diff.
+	OriginalPathsOnly bool
```

Comment:
// IgnoreNewPaths specifies that only paths that are set in the original value are
// considered in the diff -- paths that exist only in the modified struct are disregarded
// by the diff. This means that the contents of the Update field of the returned
// Notification will only contain modifications to the original value.

How about this name? I think its meaning is clearer.

Reply:
Done.